flamedisx/er_nr_base.py

```diff
@@ -659,10 +674,12 @@ class ERSource(LXeSource):
     def _single_spectrum(self):
         """Return (energies in keV, rate at these energies),
         """
+        fac = 7.
```

Comment:
???

Reply:
Extends max input ER energy to 7*10 keV and keeps the number of bins constant at 1000 bins per 10 keV. Changed variable name from `fac` to `max_e`.